packages/jest-preset/package.json

```diff
@@ -35,22 +35,22 @@
     "regenerator-runtime": "^0.13.3"
   },
   "peerDependencies": {
-    "babel-jest": "^26.0.0",
-    "jest": "^26.0.0",
+    "babel-jest": "^26.0.0 || ^27.0.0",
+    "jest": "^26.0.0 || ^27.0.0",
     "react": "^16.8.0 || ^17.0.0",
     "react-dom": "^16.8.0 || ^17.0.0",
     "react-helmet-async": "^1.0.4",
     "react-router-dom": "^5.1.0",
-    "ts-jest": "^26.0.0"
+    "ts-jest": "^26.0.0 || ^27.0.0"
```

Comment:
We should have `^27.0.3` here: https://github.com/kulshekhar/ts-jest/issues/2650#issuecomment-855832361

Reply:
Yep, you're absolutely right.